geospace-lib/src/main/scala/com/socrata/geospace/lib/shapefile/ShapeFileReader.scala

```diff
@@ -0,0 +1,46 @@
+package com.socrata.geospace.lib.shapefile
+
+import java.io.{IOException, File}
+
+import com.socrata.geospace.lib.errors.InvalidShapefileSet
+import org.geoscript.projection._
+import org.geotools.factory.Hints
+import org.geotools.referencing.ReferencingFactoryFinder
+
+
+trait ShapeReader{
```

Comment:
Add Scaladoc for the trait and all methods that need to be implemented for the trait.